pkg/bmc/bmc.go

```diff
@@ -747,8 +747,8 @@ func (bmc *BMC) SetSystemBootOrderReferences(bootOrderReferences []string) error
 	return system.SetBoot(newBoot)
 }
 
-// CreateCLISSHSession creates a ssh Session to the host.
-func (bmc *BMC) CreateCLISSHSession() (*ssh.Session, error) {
+// createCLISSHClient creates a ssh Session to the host.
+func (bmc *BMC) createCLISSHClient() (*ssh.Client, error) {
```

Comment:
since the function is now unexported, could you please move it to the bottom of the file after the exported functions?

Reply:
Fixed.